### PR TITLE
Currency: Created build script and build_info.json for keyring

### PR DESCRIPTION
**Pull request by kiranNukal**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: k/keyring/keyring_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ 
+# Install dependencies
+yum install -y --allowerasing yum-utils git gcc gcc-c++ make curl python3 python3-pip python3-devel openssl-devel pkg-config
+ 
+# Install Rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source "$HOME/.cargo/env"
+ 
+# Create and activate Python virtual environment
+python3 -m venv keyring-env
```
Comment (shubham-dayma-ibm) [inline k/keyring/keyring_ubi_9.3.sh:35]: @saiNammi why we required venv?

File: k/keyring/keyring_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+# Install additional Python packages
+pip install build pytest
+ 
+# Download and extract the keyring source tarball
+curl -O $TARBALL_URL
+tar -xvzf keyring-25.2.0.tar.gz
+cd keyring-25.2.0
+ 
+# Install the keyring package in editable mode
+pip install -e .
```
Comment (shubham-dayma-ibm) [inline k/keyring/keyring_ubi_9.3.sh:50]: Try building package first `python3 -m build;` and then installing it `pip install .` This way we might not need to install the package in editable mode

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]

File: k/keyring/keyring_ubi_9.3.sh (hunk 1 above)
Comment (kiranNukal) [reply k/keyring/keyring_ubi_9.3.sh]: Initially when I was working on ubuntu machine I have created separate env for every package which i was working on.
I have removed that thing and updated script.

File: k/keyring/keyring_ubi_9.3.sh (hunk 2 above)
Comment (kiranNukal) [reply k/keyring/keyring_ubi_9.3.sh:46]: ok

**Comment by nikhil-kalbande** [comment]
@shubham-dayma-ibm @kiranNukal  - Are we good to review and merge this Pr?

**Comment by kiranNukal** [comment]
created new branch from fork

**Pull request closed without merging**